src/Jackett.Common/Indexers/HDSpace.cs

```diff
@@ -102,90 +96,93 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
             var releases = new List<ReleaseInfo>();
+            var queryCollection = new NameValueCollection
+            {
+                {"active", "0"},
+                {"options", "0"},
+                {"category", string.Join(";", MapTorznabCapsToTrackers(query))}
+            };
 
             var searchString = query.GetQueryString();
-            var searchUrl = SearchUrl;
-            var queryCollection = new NameValueCollection();
-            queryCollection.Add("active", "0");
-            queryCollection.Add("options", "0");
-            queryCollection.Add("category", string.Join(";", MapTorznabCapsToTrackers(query)));
-
             if (!string.IsNullOrWhiteSpace(searchString))
-            {
                 queryCollection.Add("search", searchString);
-            }
 
-            searchUrl += queryCollection.GetQueryString();
-
-            var response = await RequestStringWithCookiesAndRetry(searchUrl);
-            var results = response.Content;
+            var response = await RequestStringWithCookiesAndRetry(SearchUrl + queryCollection.GetQueryString());
 
             try
             {
-                CQ dom = results;
-                var rows = dom["table.lista > tbody > tr"];
+                //TODO remove csQuery -> AngleSharp hints when all trackers are converted
```

Comment:
Please, remove all code related to csQuery. Related changes must be done in the same PR.
I will review all PRs ASAP